plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -48,6 +49,13 @@
     @VisibleForTesting
     static final DataSize DEFAULT_DATA_FILE_CACHE_SIZE = DataSize.succinctBytes(Math.floorDiv(Runtime.getRuntime().maxMemory(), 10L));
 
+    // The required reader and writer versions used by tables created by Trino
```

Comment:
Now that we have CDC this comment isn't really right, let's remove it. You could rephrase it to say that tables cannot be created with versions lower than them.